app/controllers/line_items_controller.rb

```diff
@@ -20,10 +23,12 @@ def bought_items
     current_order_cycle.items_bought_by_user(spree_current_user, current_distributor)
   end
 
-  # Override default which just redirects
-  # See Spree::BaseController and Spree::Core::ControllerHelpers::Auth
   def unauthorized
-    render text: '', status: 403
+    render nothing: true, status: 401 and return
+  end
+
+  def not_found
+    render nothing: true, status: 404 and return
   end
```

Comment:
See my most recent post on discourse. Trying to think about how we structure an API if we're going to build one. Completely take your point, but I'm reluctant to start building something until we have agreement on direction.